src/rules/use-next-tooltip.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'recommends the use of @primer/react/next Tooltip component',
+      category: 'Best Practices',
+      recommended: true,
+    },
+    fixable: null,
```

Comment:
I didn't add a fix option that updates the path from `@primer/react` to `@primer/react/next` because most of the time, Tooltip component itself will require updates as well. Let me know if you have a thought though!

Reply:
I believe its still ok to fix. Any lessening of manual work is welcome right?

Reply:
I think so! My only concern is that if we provide a fix to update the path, would it give an impression that "you are done and good to go"? Because after fixing the path, in many cases Tooltip will need an update too (i.e. `aria-label` →  `text` or adding `type="description"` etc)

Reply:
ok cool 👍 we can add it if someone requests it

Reply:
>  in many cases Tooltip will need an update too (i.e. aria-label → text or adding type="description" etc)

Will the typescript compiler warn about those issues? 

Bonus: Can aria-label → text also be fixed by the linter? 😇

Reply:
(We chatted about these questions in our call with Sid, writing here for visibility)

>Will the typescript compiler warn about those issues?

It does now :) With making the text prop required. https://github.com/primer/react/pull/3925

>Bonus: Can aria-label → text also be fixed by the linter? 😇

😇 I'm working on it to see if it is feasible to update the path as well as aria-label > text in one rule.

If anyone has any idea, thoughts, I'd love to hear 🙏🏻

Reply:
Looks like it worked